.pre-commit-config.yaml

```diff
@@ -1,13 +1,60 @@
 repos:
 -   repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: v2.3.0
+    rev: v4.6.0
     hooks:
     -   id: check-yaml
     -   id: check-json
     -   id: end-of-file-fixer
     -   id: trailing-whitespace
+        exclude: |
+          (?x)^(
+              src/core_codemods/docs/.*|
```

Comment:
I think you copied quite a bit of this and none of these locations exist in this repo, same for the other excludes, the mypy typing mocks that we may not be using